config.toml

```diff
@@ -47,140 +55,323 @@ enableRobotsTXT = true
     url = "/resources#surveys"
     parent = "Resources"
     weight = 2
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Infographics"
     url = "/resources#infographics"
     parent = "Resources"
     weight = 3
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Datasheets"
     url = "/resources#datasheets"
     parent = "Resources"
     weight = 4
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Presentations"
     url = "/resources#presentations"
     parent = "Resources"
     weight = 5
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Compatible Products"
     url = "/compatibility/"
     weight = 6
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Adopt A Spec"
     parent = "Resources"
     url = "/community/adopt-a-spec/"
     weight = 7
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Committees"
     url = "/committees"
     parent = "Resources"
     weight = 8
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Newsletter Archives"
     url = "/newsletter/"
     parent = "Resources"
     weight = 9
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Meeting Minutes"
     url = "/meeting_minutes/"
     parent = "Resources"
     weight = 10
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Blog"
     url = "https://Jakartablogs.ee"
     parent = "Resources"
     weight = 11
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Tech Talk Meet-ups"
     url = "/resources#meetups"
     parent = "Resources"
     weight = 12
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Jakarta EE YouTube Channel"
     url = "https://www.youtube.com/channel/UC4M7h5b6elu9VlzjLfzuXyg"
     parent = "Resources"
     weight = 13
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Jakarta EE Specification Projects"
     url = "https://projects.eclipse.org/list-of-projects?combine=jakarta&field_project_techology_types_tid=1102&field_state_value_2=All"
     parent = "Resources"
     weight = 14
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Jakarta EE Community Folder"
     url = "https://drive.google.com/open?id=1kJFh6EIpOyAWP3JkymYZXOK311WM2HXm"
     parent = "Resources"
     weight = 15
     
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Jakarta EE Community Calendar"
     url = "/resources#calendar"
     parent = "Resources"
     weight = 16
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Starter for Jakarta EE"
     url = "https://start.jakarta.ee/"
     parent = "Resources"
     weight = 17
     
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "JakartaOne Livestream"
     url = "https://jakartaone.org/"
     parent = "Resources"
     weight = 18
+
+# for Chinese Version below
+
+[languages.zh.Params]
+  google_tag_manager = "GTM-5WLCZXC"
+  description = "Jakarta® Enterprise Edition (EE) 是云原生的未来. Jakarta® EE 开源软件驱动云原生创新, 让企业应用更加现代化，并同时保护Java EE的现有投资."  
+  subtitle = "云原生Java的新家园"
+  seo_title_suffix = " | Eclipse 基金会"
```

Comment:
As "Eclipse Foundation" is as a whole the trademarked company name, we shouldn't be changing half into another language. This should be left as is, as "Eclipse Foundation" itself is a brand